src/develop/imageop.c

```diff
@@ -844,6 +845,7 @@ dt_iop_module_t *dt_iop_gui_duplicate(dt_iop_module_t *base, gboolean copy_param
                           expander, g_value_get_int(&gv) + pos_base - pos_module + 1);
     dt_iop_gui_set_expanded(module, TRUE, FALSE);
     dt_iop_gui_update_blending(module);
+    darktable.gui->reset = 0;
```

Comment:
darktable.gui->reset = reset;

To properly reset the state as it was entering this routine.